### ISSUE TITLE
User log

### ISSUE THREAD

**Issue by boyeln** [RadioRevolt/DABelFish#14]
Add a log of a users activity (admin feature). This should be placed in the left-side panel on the user-page (/client/pages/user).
Use a table to display paginated user activity. Either find a clever way of merging user logs (create user, change password ect) and dabText logs, og display them separately. Use the log on the home page as reference (/client/pages/home/home.html).

**Comment by boyeln** [comment]
Something like this in the panel-body:

``` html
<table class="table table-hover table-responsive">
  <thead>
    <tr>
      <th>What</th>
      <th>When</th>
    </tr>
  </thead>
  <tbody>
    {{#each logs}}
      <tr>
        <td>{{what}}</td>
        <td>{{formatDate when}} by {{createdBy}}</td>
      </tr>
    {{/each}}
  </tbody>
</table>
<nav>
  <ul class="pager">
    <li class="previous {{paginationNewerDisabled}}"><a href="#" id="pagination-newer"><span aria-hidden="true">&larr;</span> Newer</a></li>
    <li class="next {{paginationOlderDisabled}}"><a href="#" id="pagination-older">Older <span aria-hidden="true">&rarr;</span></a></li>
  </ul>
</nav>
```

And then some clever helper to create the logs and session variables to keep track of the pagination.

**Comment by boyeln** [comment]
Fixed in 29ba7b0b1f0c54022568e6431ba91bbc90981fa8
